lib/Conversion/TorchToLinalg/Reduction.cpp

```diff
@@ -228,14 +233,176 @@ static Value createLinalgPayloadCalculationForReduceOp(
       if (intType.isSigned())
         return b.create<arith::MaxSIOp>(loc, self, result);
     }
+  } else if (isa<AtenLinalgVectorNormOp>(op)) {
+    // This creates payload for only the first of the two linalg.generic ops.
+    // TODO: Short-circuit operations if `ord` is zero or one.
+    auto elem = payloadArgs[0];
+    auto result = payloadArgs[1];
+    auto self = convertScalarToDtype(b, loc, elem, resultElementType);
+    auto abs = b.create<math::AbsOp>(loc, self);
+    auto ord = convertScalarToDtype(b, loc, operands[1], resultElementType);
```

Comment:
Sadly, I don't think we can use `op.ord()` in place of `operands[1]`, because `op.ord()` is of type `torch.int`, whereas we need the ord value after type conversion.